src/libYARP_dev/src/IPositionControlImpl.cpp

```diff
@@ -160,7 +160,14 @@ bool ImplementPositionControl::checkMotionDone(const int n_joint, const int *joi
 
 bool ImplementPositionControl::checkMotionDone(bool *flag)
 {
-    return iPosition->checkMotionDoneRaw(flag);
+    bool flags_tmp[nj];
+    bool ret = iPosition->checkMotionDoneRaw(flags_tmp);
+    for(int i=0; i<nj; i++)
+    {
+        flag[i] = flags_tmp[castToMapper(helper)->toHw(i)];
+    }
+    delete []flags_tmp;
```

Comment:
`flag_tmp` has not dynamically allocated, coverity is right, this `delete` has to be removed.

Reply:
now flags_tmp is allocated dinamically: before it was not declared in standard way.
It was fault. sorry! see https://github.com/robotology/yarp/pull/1804/commits/8cf860aebc8e4cb30402b2501622937ee7928560